src/main/asciidoc/index.adoc

```diff
@@ -143,27 +143,35 @@ You can change the control, e.g you can send a response immediately to the user-
 {@link examples.HttpProxyExamples#immediateResponse}
 ----
 
-==== Header interceptor
+==== Head interceptor
```

Comment:
we should mention that each operation of the builder can be called multiple times

Reply:
Done